freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/repair/changeprimarygw/action/ChangePrimaryGatewayActions.java

```diff
@@ -150,7 +154,11 @@ protected void doExecute(ChangePrimaryGatewayContext context, StackEvent payload
                 Stack stack = context.getStack();
                 SuccessDetails successDetails = new SuccessDetails(stack.getEnvironmentCrn());
 
-                stackUpdater.updateStackStatus(stack.getId(), getChangePrimaryGatewayCompleteStatus(variables), "Finished changing the primary gateway");
+                if (isFinalChain(variables)) {
```

Comment:
this `if` can be combined with the `if` starting on line 162.

Reply:
Thanks a lot for the suggestion. I've combined the two "if"s and tested the flow again. 

After repair, the status check updates it to available. 
<img width="590" alt="Screen Shot 2020-10-19 at 8 31 31 PM" src="https://user-images.githubusercontent.com/67844271/96529180-17bbde00-124a-11eb-9071-6afc5c4843cc.png">


<img width="979" alt="Screen Shot 2020-10-19 at 8 32 12 PM" src="https://user-images.githubusercontent.com/67844271/96529225-33bf7f80-124a-11eb-8f9f-e0cd3f5c5cfb.png">